srtp_writer_future.go

```diff
@@ -66,7 +78,7 @@ func (s *srtpWriterFuture) ReadContext(ctx context.Context, b []byte) (n int, er
 		return value.(*srtp.ReadStreamSRTCP).ReadContext(ctx, b)
 	}
 
-	if err := s.init(ctx); err != nil || s.rtcpReadStream.Load() == nil {
+	if err := s.init(ctx, false); err != nil || s.rtcpReadStream.Load() == nil {
```

Comment:
I guess it should be true for context controlled usage and false for no-context usage. It seems to be true for this case?

Reply:
or all `init()` call in Read/Write/ReadRTP/WriteRTP should set false.